.github/actions/orca/orca-notebook-xshards-action/action.yml

```diff
@@ -27,6 +27,13 @@ runs:
         pip install -i ${GONDOLIN_PIP_MIRROR} --trusted-host ${GONDOLIN_TRUSTED_HOST} jupyter
         pip install -i ${GONDOLIN_PIP_MIRROR} --trusted-host ${GONDOLIN_TRUSTED_HOST} seaborn
         pip install -i ${GONDOLIN_PIP_MIRROR} --trusted-host ${GONDOLIN_TRUSTED_HOST} pyarrow 
+        pip install -i ${GONDOLIN_PIP_MIRROR} --trusted-host ${GONDOLIN_TRUSTED_HOST} ray[default]==1.9.2
+        pip install -i ${GONDOLIN_PIP_MIRROR} --trusted-host ${GONDOLIN_TRUSTED_HOST} protobuf==3.15.3
```

Comment:
need to check which example uses protobuf

Reply:
fix issue
~~~
++ ray stop -f
Traceback (most recent call last):
  File "/opt/conda/envs/orca-tutorial-xshards-py37/bin/ray", line 5, in <module>
    from ray.scripts.scripts import main
  File "/opt/conda/envs/orca-tutorial-xshards-py37/lib/python3.7/site-packages/ray/__init__.py", line 91, in <module>
    import ray._raylet  # noqa: E402
  File "python/ray/_raylet.pyx", line 104, in init ray._raylet
  File "/opt/conda/envs/orca-tutorial-xshards-py37/lib/python3.7/site-packages/ray/exceptions.py", line 5, in <module>
    from ray.core.generated.common_pb2 import RayException, Language, PYTHON
  File "/opt/conda/envs/orca-tutorial-xshards-py37/lib/python3.7/site-packages/ray/core/generated/common_pb2.py", line 37, in <module>
    create_key=_descriptor._internal_create_key),
  File "/opt/conda/envs/orca-tutorial-xshards-py37/lib/python3.7/site-packages/google/protobuf/descriptor.py", line 755, in __new__
    _message.Message._CheckCalledFromGeneratedFile()
TypeError: Descriptors cannot not be created directly.
If this call came from a _pb2.py file, your generated code is out of date and must be regenerated with protoc >= 3.19.0.
If you cannot immediately regenerate your protos, some other possible workarounds are:
 1. Downgrade the protobuf package to 3.20.x or lower.
 2. Set PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python (but this will use pure-Python parsing and will be much slower).

~~~
test link
https://github.com/intel-analytics/BigDL/actions/runs/3872116230/jobs/6600594065

Reply:
![image](https://user-images.githubusercontent.com/21994046/212608099-5efe35dc-3b72-4ffe-acef-1cb14893c8b0.png)
Seems installing tf1.15.0 will install the latest protobuf, which conflict with ray? @hamham223 Please take a look.